### PR TITLE
Updated for New react course

### PR DESCRIPTION
**Pull request by chezzoba**

### CODE REVIEW

File: README.md
Comment (adamblanchard) [inline README.md:172]: I know it wasn't your typo but while you're here can you fix "TypScript" please 😅 🙏

**Comment by adamblanchard** [comment]
Thanks for owning it @chezzoba! I don't know enough to review the code, so i'll leave that up to others.

My questions:

1. Should we keep support for both Vite _and_ Next? As I understand vite was added here for it's simplicity and the ability to "not really care much about it" when building quick apps. In the aim of keeping this template generic and flexible, and usable before people have reached the react module, I'm wondering if Next should be an additional option rather than a replacement?

I think a problem is we are not really 100% aligned on all the tools we should be using and teaching throughout the curriculum. Deciding that first would probably help, but to avoid blocking this change with a big discussion first, wondering what the best thing to do in the meantime is.

2. Should we merge this as part of merging the new react module? For consistency and also in case anything else changes after feedback?

**Comment by chezzoba** [comment]
Hi Adam,

Short answer: (1) Sure, we can keep support for both and (2) yes, let's! Thanks a lot!

I have simply made the same app, but with a similar setup to what students experience in class. In my opinion, Next.js is easier to understand and get started with compared to Vite if you don't already know vanilla React beforehand. However, I don't have a strong opinion on this and it is completely fine if we include both. Just thought we wouldn't want to confuse the students 😀

We teach Next as it is the official React framework, and what React recommends on their [getting started page](https://react.dev/learn/start-a-new-react-project). However, I understand that many people have many different ways of starting React projects, which is why I have left it open for students to choose their own for their projects (eg. Meal sharing app) in React 2. Leaving both available might be a nice idea for this. However, right now only having the Vite one is confusing students.

Thank you so much for your support in this!

**Review by adamblanchard — APPROVED** [review]
Nice! Approved.

Please let some other more knowledgable devs review the code too :-)

I'll merge it after that 👌

**Review by shpomp — APPROVED** [review]

**Pull request merged**
